independent-projects/bootstrap/maven-resolver/pom.xml

```diff
@@ -56,6 +56,24 @@
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-embedder</artifactId>
+            <exclusions>
+                <exclusion><!-- let's prefer the plexus-utils pulled by maven-resolver-provider -->
+                    <groupId>org.codehaus.plexus</groupId>
+                    <artifactId>plexus-utils</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>javax.annotation</groupId>
+                    <artifactId>javax.annotation-api</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>javax.inject</groupId>
+                    <artifactId>javax.inject</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>javax.enterprise</groupId>
+                    <artifactId>cdi-api</artifactId>
```

Comment:
I need to take a closer look at this, given that it should have been removed in 3.8.3: https://issues.apache.org/jira/browse/MNG-7045

Also, I think the remaining exclusions need to be defined in the bootstrap BOM, not here...?

Reply:
As I thought: not needed anymore!

I have added two more commits locally that I will give a spin in fork's CI. If all goes to plan I will update this branch/PR.

@jorsol I hope you don't mind if I amend your first commit?

Reply:
Sure @famod, I don't mind if you amend my commit.

Reply:
Ok thanks, done!